build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
 
                 // TODO: Unreleased dependencies.
                 // These must already be installed in the local maven repository.
-                netty: 'io.netty:netty-codec-http2:4.1.0.Beta5-SNAPSHOT',
+                netty: 'io.netty:netty-codec-http2:4.1.0.Beta6-SNAPSHOT',
```

Comment:
Can't someone who needs netty/netty@dca6cea just depend on it in their application?